Framework/DataHandling/test/LoadPLNTest.h

```diff
@@ -0,0 +1,82 @@
+#ifndef LOADPLNTEST_H_
+#define LOADPLNTEST_H_
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidAPI/Run.h"
+#include "MantidDataHandling/LoadPLN.h"
+
+using namespace Mantid::API;
+using namespace Mantid::Kernel;
+using namespace Mantid::DataHandling;
+using namespace Mantid::DataObjects;
+
+class LoadPLNTest : public CxxTest::TestSuite {
+public:
+  void test_load_pln_algorithm_init() {
+    LoadPLN algToBeTested;
+
+    TS_ASSERT_THROWS_NOTHING(algToBeTested.initialize());
+    TS_ASSERT(algToBeTested.isInitialized());
+  }
+
+  void test_load_pln_algorithm() {
+    LoadPLN algToBeTested;
+
+    if (!algToBeTested.isInitialized())
+      algToBeTested.initialize();
+
+    std::string outputSpace = "LoadPLNTest";
+    algToBeTested.setPropertyValue("OutputWorkspace", outputSpace);
+
+    // should fail because mandatory parameter has not been set
+    TS_ASSERT_THROWS(algToBeTested.execute(), std::runtime_error);
+
+    // missing event file - should fail execution
+    std::string inputFile = "PLN0044464.hdf";
+    algToBeTested.setPropertyValue("Filename", inputFile);
+    TS_ASSERT(!algToBeTested.execute());
+
+    // should succeed now
+    algToBeTested.setPropertyValue("Filename", inputFile);
+    algToBeTested.setPropertyValue("BinaryEventPath", "./PLN0044464.bin");
+    algToBeTested.setPropertyValue("CalibrateTOFBias", "1");
+    TS_ASSERT_THROWS_NOTHING(algToBeTested.execute());
+    TS_ASSERT(algToBeTested.isExecuted());
+
+    // get workspace generated
+    MatrixWorkspace_sptr output =
+        AnalysisDataService::Instance().retrieveWS<MatrixWorkspace>(
+            outputSpace);
+
+    // check number of histograms
+    TS_ASSERT_EQUALS(output->getNumberHistograms(), 12808);
+    double sum = 0.0;
+    for (size_t i = 0; i < output->getNumberHistograms(); i++)
+      sum += output->readY(i)[0];
```

Comment:
readY is deprecated, use y instead?

Reply:
I started about 6mths ago and the priority was to get some additional instruments onto Mantid. The plan is to refactor the code, without changing functinality, to increase re-use between EMU, PLN and BBY later this year.  I will look at re-using as much of the Mantid support functions as possible. 
readY() changed to y().

Reply:
I followed the existing ANSTO logic, such as LoadBBY for Bilby, but I accept that it may be more historical than logical.

Reply:
Ok for me, if this naming logic is used ANSTO internally. Then, your clients may not experience problems using Mantid